test-automation/src/main/java/database/DatabaseSetup.java

```diff
@@ -0,0 +1,19 @@
+package database;
+
+public class DatabaseSetup
+{
+    public static String createConnectionString(String environment) throws Exception
+    {
+        switch (environment.toUpperCase())
+        {
+            case "QA":
+                return "";
+
+            case "DEV":
+                return "jdbc:oracle:thin:@devdb.bcgov:1521/deva";
+
```

Comment:
DB url should come from environment values as well. This should not be committed to the repo